completion.go

```diff
@@ -0,0 +1,177 @@
+package kong
+
+import (
+	"github.com/posener/complete"
+	"github.com/posener/complete/cmd/install"
+)
+
+// CompletionOptions options for shell completion.
+type CompletionOptions struct {
+	// RunCompletion determines whether to respond to shell completion requests. default: false
+	// When RunCompletion is false, all other options in CompletionOptions are ignored.
+	RunCompletion bool
```

Comment:
Unless there's some downside I'm unaware of, I'd prefer this to either be removed altogether or default to true.